### PR TITLE
docs: Soft expect example + tobeAttached

### PR DESCRIPTION
**Pull request by debs-obrien**
- add an example for soft expect
- add toBeAttached to table

### CODE REVIEW

File: docs/src/test-webserver-js.md [hunk 1]
```diff
@@ -18,6 +18,8 @@ export default defineConfig({
     command: 'npm run start',
     url: 'http://127.0.0.1:3000',
     reuseExistingServer: !process.env.CI,
+    stdout: 'ignore',
```
Comment (aslushnikov) [inline docs/src/test-webserver-js.md:21]: this is part of https://github.com/microsoft/playwright/pull/23474 - so probably should be reverted from this PR

**Review by aslushnikov — APPROVED** [review]

**Comment by aslushnikov** [comment]
Note: you mentioned "toBeAttached" in the table, but I didn't find it

**Comment by debs-obrien** [comment]
> Note: you mentioned "toBeAttached" in the table, but I didn't find it

we don't seem to give many examples for our assertions. I guess we presume people either understand them or will read api docs so we should discuss what is best here as in just table or add examples for them all

**Comment by debs-obrien** [comment]
as discussed in meeting we will leave table with link to api docs for example

**Pull request merged**